ci-operator/step-registry/telco5g/jobs-router/telco5g-jobs-router-commands.sh

```diff
@@ -50,6 +50,7 @@ elif [[ "$JOB_TYPE" == "periodic" ]]; then
     echo "export T5_JOB_RELEASE_IMAGE='$IMG_URL'" >> "$MAINENV"
     echo "export KCLI_PARAM='-P tag=${T5CI_VERSION} -P version=nightly'" >> "$MAINENV"
     echo "export SNO_PARAM='-t ${T5CI_VERSION} -r nightly'" >> "$MAINENV"
+    echo "export REPO_OWNER=''" >> "$MAINENV"
```

Comment:
Let's add also `REPO_NAME` extraction

Reply:
added empty value for `REPO_NAME` for periodic case.

Reply:
I suggest to move it to a un-conditional part in lines 13-21, so in case of nightly jobs or any other it won't fail

Reply:
done